packages/sdk/typedoc.json

```diff
@@ -7,8 +7,14 @@
     "docs/Why-Vincent.md",
     "docs/Concepts.md",
     "docs/Developers/Quick-Start.md",
-    "docs/Developers/Creating-Policies.md",
-    "docs/Developers/Creating-Tools.md",
+    "docs/Developers/Getting-Started.md",
+    "docs/Developers/App-Agent-Developers/Getting-Started.md",
+    "docs/Developers/Tool-Developers/Getting-Started.md",
+    "docs/Developers/Tool-Developers/Creating-Tools.md",
+    "docs/Developers/Policy-Developers/Getting-Started.md",
+    "docs/Developers/Policy-Developers/Creating-Policies.md",
+    "docs/Developers/AI-Integration.md",
+    "docs/Developers/MCP.md",
```

Comment:
MCP.md replaced AI-Integration.md and I don't see them modified here. Why this change?